src/app/page.cljs

```diff
@@ -11,13 +11,14 @@
   {:title "CopyCat",
    :icon "http://cdn.tiye.me/logo/mvc-works.png",
    :ssr nil,
-   :inline-html nil,
-   :inline-styles [(slurp "entry/main.css")]})
+   :inline-html nil})
 
 (defn dev-page []
   (make-page
    ""
-   (merge base-info {:styles ["http://localhost:8100/main.css"], :scripts ["/main.js"]})))
+   (merge
+    base-info
+    {:styles ["http://localhost:8100/main.css" "entry/main.css"], :scripts ["/client.js"]})))
```

Comment:
这里一定要强关联 path 和 port 吗 有没有更好的形式？

Reply:
这个是开发环境的配置, `localhost:8100` 是我本地的 CDN 服务. 发布到个人服务器的代码会直接用 http://cdn.tiye.me/ 的资源.

目前没想到更好的形式. 有建议吗?

Reply:
写在配置文件里？

Reply:
要在上游的仓库里面设计一下方案了... https://github.com/mvc-works/calcit-workflow/blob/master/src/app/page.cljs#L18

我有个专门的 Nginx 配置来提供这个文件,

```
=>> pwd
/usr/local/etc/nginx/servers
=>> cat fonts.conf

server {
  listen      8100;
  server_name localhost;
  location / {
    root /Users/chen/repo/tiye/favored-fonts/cdn/;
    autoindex on;
    index index.html;

    if ($request_method = 'OPTIONS') {
      add_header 'Access-Control-Allow-Origin' '*';
      add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';

      add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
      add_header 'Access-Control-Max-Age' 1728000;

      add_header 'Content-Type' 'text/plain; charset=utf-8';
      add_header 'Content-Length' 0;
      return 204;
    }
    if ($request_method = 'POST') {
        add_header 'Access-Control-Allow-Origin' '*';
        add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
        add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
        add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
    }
    if ($request_method = 'GET') {
        add_header 'Access-Control-Allow-Origin' '*';
        add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
        add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
        add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
    }
  }
}
```

这个跟我的个人开发环境绑定得挺死的.